recipe/meta.yaml

```diff
@@ -22,11 +23,11 @@ requirements:
   run:
     - python
     - cloudpickle
-    - ipykernel >=6.16.1,<7.0.0
-    - ipython >=7.31.1,<9.0.0
+    - ipykernel >=6.23.2,<7.0.0
```

Comment:
```suggestion
    - ipykernel >=6.23.2,<7
```

Reply:
7.0.0 is perfectly fine.

Reply:
To clarify why, upstream uses `>=6.23.2,<7`. In general, it's better to just use what upstream uses if it works to avoid unnecessary diffs.

Reply:
ok cool @JeanChristopheMorinPerso , I will resolve this then